src/_includes/layouts/events.njk

```diff
@@ -0,0 +1,88 @@
+---
+layout: layouts/fullwidth.njk
```

Comment:
This will make the page title based on the event name e.g. "application-deleted - OPG Event Store"
```suggestion
layout: layouts/fullwidth.njk
eleventyComputed:
    title: "{{ name }}"
```
You can do the same with services, domains etc.